src/benchmark_register.cc

```diff
@@ -159,6 +159,7 @@ bool BenchmarkFamilies::FindBenchmarks(
         instance.use_manual_time = family->use_manual_time_;
         instance.complexity = family->complexity_;
         instance.complexity_lambda = family->complexity_lambda_;
+        instance.statistics = &family->statistics_;
```

Comment:
This one would break too:
https://github.com/google/benchmark/blob/d70417994a3c845c49c4443e92b26a52b320a759/src/benchmark_register.cc#L150